spec/lib/player_spec.rb

```diff
@@ -0,0 +1,104 @@
+require './lib/scrabble/player'
+
+describe "Player class" do
+  let(:patricia) {
+    x = Scrabble::Player.new("Patricia")
+    x.play("good") # 6 pts
+    x.play("game") # 7 pts
+    x.play("girl") # 5 pts
+    return x
+  }
+
```

Comment:
Normally, `let` blocks are just for assigning objects to variables (in this case, `patricia` as an instance of `Scrabble::Player`. If you want certain methods to be called before your tests to set up the situation you want to test, try using a `before` hook. https://www.relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks